oracle/tests/conftest.py

```diff
@@ -57,3 +117,17 @@ def dd_environment():
     ],
     'env_vars': {'LD_LIBRARY_PATH': '/opt/oracle/instantclient_19_3'},
 }
+
+E2E_METADATA_JDBC_CLIENT = {
+    # Since we don't include Oracle instantclient to `LD_LIBRARY_PATH` env var,
+    # the integration will fallback to JDBC client
+    'use_jmx': True,  # Using jmx to have a ready to use java runtime
+    'start_commands': [
+        'mkdir /opt/oracle',
+        'apt-get update',
+        'apt-get install libaio1 unzip',
+        'curl -o /opt/oracle/instantclient.zip '
+        'https://storage.googleapis.com/datadog-integrations-lab/instantclient-basiclite-linux.x64-19.3.0.0.0dbru.zip',
+        'unzip /opt/oracle/instantclient.zip -d /opt/oracle',
```

Comment:
Why do you need this for JDBC?

Reply:
Needed for the `ojdbc8.jar`: https://github.com/DataDog/integrations-core/pull/5179/files#diff-33d921be5cdcd22426a6ee8d8d32a4c0R51

Maybe I can just extract this exact file.

Reply:
@therve Fixed :) 

Removed libaio1 and add comment about why we need the instant client.